concordium-consensus/src/Concordium/KonsensusV1/TreeState/Implementation.hs

```diff
@@ -240,7 +240,17 @@ data SkovData (pv :: ProtocolVersion) = SkovData
       _currentTimeoutMessages :: !(Option TimeoutMessages),
       -- |The 'QuorumMessage's for the current 'Round'.
       -- This should be cleared whenever the consensus runner advances to a new round.
-      _currentQuorumMessages :: !QuorumMessages
+      _currentQuorumMessages :: !QuorumMessages,
+      -- |Whether consensus is shutdown.
+      -- This is @True@ when the epoch where the protocol update "took effect"
+      -- (with respect to the effective time of the protocol update).
```

Comment:
What does it mean that an epoch "took effect"? This needs to be clearer. I think this means that:
1. The protocol update was effective in the trigger block in this epoch.
2. The trigger block is finalized.

Reply:
Added docs.